frontend/src/lib/router.tsx

```diff
@@ -182,41 +182,41 @@ const defaultRoutes: {
     component: () => <NodeDetails />,
   },
   storageClasses: {
-    path: '/storage/classes',
+    path: '/storageclasses',
```

Comment:
What exactly are these fixing? I think changing a long standing URL may represent a breaking change.

Reply:
it's inconsistent with the rest of the paths, every other route is using /apiName/:namespace/:name